src/utils/messages.py

```diff
@@ -44,6 +44,20 @@ class ConfigurationMessage:
     updated: str = "Configuration updated"
 
 
+@dataclass
+class MobilityGroupsMessages:
+    """
+    Messages used in endpoint responses for mobility groups
+    """
+    found: str = "Mobility Groups found"
+    not_found: str = "Mobility Groups not found"
+    exist: str = 'Mobility Groups exist'
+    created: str = "Mobility Groups has been created"
+    updated: str = "Mobility Groups updated"
+    not_mobility_group_entry: str = "Mobility Groups not entry"
+    not_distribution_entry: str = "Distribution not entry"
```

Comment:
Qué significa "not entry" ... ?

Revisen por favor este mensaje.

Reply:
Sorry, debería ser "not entered", trato de decir que no ha ingresado esa información. Lo corrijo.